tools/CheckFiles.sh

```diff
@@ -219,8 +219,3 @@ if ! find . \
      ! -name deploy_key.enc \
      -print0 \
         | run_checks "${standard_checks[@]}" "${ci_checks[@]}"
-then
-    echo "This script can be run locally from any source dir using:"
-    echo "SPECTRE_ROOT/tools/CheckFiles.sh"
```

Comment:
Are you saying it just doesn't run from subdirectories anymore? I occasionally use the script from the top-level dir and that's working fine. Perhaps leave the message and just remove the "from any source dir"?

Reply:
Yes, it still works at top-level.

CI already prints out the command (and we can't screw that one up if we rename the file).